src/vue/backend/src/router/index.js

```diff
@@ -1,22 +1,171 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
-import Home from '../views/Home.vue'
 
 Vue.use(VueRouter)
 
 const routes = [
     {
-        path: '/',
-        name: 'Home',
-        component: Home
+        path: '/import',
+        children: [
+            {
+                path: '/listings',
+                children: [
+                    {
+                        path: 'imported',
+                        name: 'Imported IDX Listings'
+                        // component
+                    },
+                    {
+                        path: '',
+                        name: 'Unimported IDX Listings'
+                        // component
+                    }
+                ]
+            },
+            {
+                path: '/agents',
+                children: [
+                    {
+                        path: 'imported',
+                        name: 'Imported Agents'
+                        // component
+                    },
+                    {
+                        path: '',
+                        name: 'Unimported Agents'
+                        // component
+                    }
+                ]
+            }
+        ]
     },
     {
-        path: '/about',
-        name: 'About',
-        // route level code-splitting
-        // this generates a separate chunk (about.[hash].js) for this route
-        // which is lazy-loaded when the route is visited.
-        component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')
+        path: '/settings',
+        name: 'Settings',
+        children: [
+            {
+                path: 'general',
+                name: 'IMPress General Settings'
+                // component
+            },
+            {
+                path: 'omnibar',
+                name: 'IMPress Omnibar Settings'
+                // component
+            },
+            {
+                path: 'listings',
```

Comment:
The child routes here (since this is part of a tab) need a path. The leftmost tab (default) should remain as an empty string.